contrib/pipelines/secadvisor/secadvisor.yml.default

```diff
@@ -1,35 +1,43 @@
+# host_id is used to reference the agent, should be unique or empty
 host_id: ""
+# list of analyzers for running REST requests
+analyzers:
+  - 127.0.0.1:8082
 pipeline:
+  # analyzer credentials for websocket connection
   analyzer:
     subscriber_url: ws://127.0.0.1:8082/ws/subscriber/flow
     subscriber_username:
     subscriber_password:
+  # classify flows into: internal, ingress, egress and other
   classify:
-    # cluster_net_masks:
-      # - 10.0.0.0/8
-      # - 172.16.0.0/12
-      # - 192.168.0.0/16
+    # list of internal cluster address ranges
+    cluster_net_masks:
+      - 10.0.0.0/8
+      - 172.16.0.0/12
+      - 192.168.0.0/16
+  # filter out flows which match a criteria
   filter:
-    # excluded_tags:
-      # - internal
-      # - other
+    # exclude the following classified tags 
+    excluded_tags:
+      - other
+  # transform to sa record structure
   transform:
     sa:
       # exclude_started_flows: true
   store:
     type: s3
     s3:
-      # -- client parames --
       endpoint: http://127.0.0.1:9000
-      region: local
-      bucket: bucket
       access_key: user
       secret_key: password
+      region: local
```

Comment:
@hunchback can you check why we have a region configuration variable ... if we nave `object_prefix` why can't we add the region as part of the `object_prefix` ? (if so need to drop this configuration key)

Reply:
better it is a separate parameter, as region my change between deployments while prefix is set according to conventiom

Reply:
There are many 'things' that change between deployment ... (for example today for SA we put the k8s GUID as part of the prefix)  .... if @orozery did not have any specific reason for that split between region and prefix ... maybe this is a mistake / leftover from some porting process ... for the general case this is really not needed. I think that this will be a separate PR but I do not see why we need this extra confusing parameter.

Reply:
@eranra - when we extent to fully support AWS Flow Logs then I will address region (as the AWS Flow Logs has a unique URL structure).